src/plugins/intel_gna/src/backend/dnn_types.hpp

```diff
@@ -96,6 +96,7 @@ typedef enum {
     kDnnDiagonalOp,
     kDnnConvolutional1dOp,
     kDnnConvolutional2dOp,
+    kDnnDWSCOp,
```

Comment:
do we need to add this to FP32 runtime ?

Reply:
I think so.